src/Suggestions.js

```diff
@@ -1,69 +0,0 @@
-import React from "react";
-import Header from "./Header";
-import App from "./App";
-import { useState, useEffect } from "react";
-
-const Suggestions = ({ mood, socialize, location }) => {
-    const [startOver, setStartOver] = useState(false);
-    const [resultHeading, setResultHeading] = useState("");
-    const [resultImg, setResultImg] = useState("");
-
-    function handleRetakeQuiz(e) {
-        e.preventDefault();
-        setStartOver(true)
-    }
-
-    useEffect(() => {
-        getData();
-    })
-
-    const getData = () => {
-        fetch('data.json',
-            {
-                headers: {
-                    'Content-Type': 'application/json',
-                    'Accept': 'application/json'
-                }
-            }).then((response) => response.json())
-            .then((data) => {
-                console.log(data);
-                let value = `${mood}${location}${socialize}`
-                console.log(value);
-                console.log(data[value].result);
-                setResultHeading(data[value].result);
-                setResultImg(data[value].source)
-            }).catch((error) => console.log("There is an error!", error))
-    }
-
-    if (startOver === false) {
```

Comment:
since `startOver` is a boolean field, you could do `if(startOver){ return <App />} else {...}`

Reply:
Makes sense. Have made the changes.